tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1522,4 +1531,42 @@ class CompletionSuite extends BaseCompletionSuite {
     ),
   )
 
+  check(
+    "class-members-trait-issue",
```

Comment:
this one is failing on the other PR's branch. Can we solve it in another way without a total rework, since now on this  branch we have the regression of not adding type parameter square brackets in here.

Reply:
perhaps, we can just remove the filter for `isNotClassOrTraitOrTheyAreValidForPos` for now to fix this regression.

Reply:
by simply adding `|| sym.isTerm` to `isNotClassOrTraitOrTheyAreValidForPos` the regression would be solved.

Reply:
I think this refactoring is fine and might be more maintainable.